src/main/java/org/dcsa/ebl/service/impl/TransportDocumentServiceImpl.java

```diff
@@ -318,7 +317,6 @@ void setIssuerOnTransportDocument(TransportDocumentTO transportDocumentTO, Carri
     shipmentEvent.setEventType(null);
     shipmentEvent.setEventClassifierCode(EventClassifierCode.ACT);
     shipmentEvent.setDocumentTypeCode(DocumentTypeCode.SHI);
-    shipmentEvent.setCarrierBookingReference(transportDocumentTO.getTransportDocumentReference());
```

Comment:
What ?!? setting carrier booking reference with the transportDocumentReference!!!
Good you deleted this line :-D